ios/Scripts/install_icons_from_zip.sh

```
@@ -0,0 +1,149 @@
+    echo "🎯 Иконки успешно установлены!"
+    echo "📱 Теперь откройте проект в Xcode:"
+    echo "   open PulsePlate.xcodeproj"
+
+    return 0
+}
+cleanup() {
+    echo "🧹 Очищаем временные файлы..."
+    rm -rf "$TEMP_DIR/extracted"
+    echo "✅ Временные файлы удалены"
+}
+ }
```

Shellcheck (0.11.0)

[error] Parsing stopped here. Is this keyword correctly matched up?

(SC1089)